src/i2c.rs

```diff
@@ -68,6 +69,15 @@ impl Mode {
             frequency: frequency.into(),
         }
     }
+
```

Comment:
Removed due to complexity of calculating frequency in case of custom, also it seems like unnecessory